ledger/sandbox/src/main/scala/com/digitalasset/platform/index/JdbcIndexer.scala

```diff
@@ -350,12 +374,14 @@ class JdbcIndexer private[index] (
   private def stateUpdateRecordTime(update: Update): Option[Instant] =
     (update match {
       case Heartbeat(recordTime) => Some(recordTime)
-      case PartyAddedToParticipant(_, _, _, recordTime) => Some(recordTime)
-      case PublicPackageUploaded(_, _, _, recordTime) => Some(recordTime)
+      case PartyAddedToParticipant(_, _, _, recordTime, _) => Some(recordTime)
+      case PublicPackageUploaded(_, _, _, recordTime, _) => Some(recordTime)
       case TransactionAccepted(_, _, _, _, recordTime, _) => Some(recordTime)
       case ConfigurationChanged(_, _) => None
       case ConfigurationChangeRejected(_, _) => None
       case CommandRejected(_, _) => None
+      case PackageUploadEntryAccepted(_, _) => None
```

Comment:
Why does the accept message not contain a record time, similar to `TransactionAccepted`?

Reply:
added recordTime to these Update events too

Reply:
heads-up that my config indexing PR adds the recordTime to the abstract base class.